### PR TITLE
SIRI-266: Ensures that fallback is added correctly in an MLSProperty

### PR DESCRIPTION
**Pull request by cschierle**
Fixes: SIRI-266

### CODE REVIEW

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java [hunk 1]
```diff
@@ -206,6 +207,14 @@ protected boolean isConsideredNull(Object propertyValue) {
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
 
+    @Override
+    public void parseValues(Object e, Values values) {
+        MultiLanguageString multiLanguageString = getMultiLanguageString(e);
+        Map<String, String> mlsMap = new HashMap<>(multiLanguageString.data());
+        mlsMap.put("fallback", values.at(0).getString());
+        setValue(e, mlsMap);
+    }
```
Comment (bwiedmann) [inline src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java:216]:
```suggestion
    public void parseValues(Object entity, Values values) {
        MultiLanguageString multiLanguageString = getMultiLanguageString(entity);
        Map<String, String> mlsMap = new HashMap<>(multiLanguageString.data());
        mlsMap.put("fallback", values.at(0).getString());
        setValue(entity, mlsMap);
    }
```

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java [hunk 2]
```diff
@@ -206,6 +207,14 @@ protected boolean isConsideredNull(Object propertyValue) {
         return values.entrySet().stream().anyMatch(entry -> Strings.isEmpty(entry.getValue()));
     }
 
+    @Override
+    public void parseValues(Object entity, Values values) {
+        MultiLanguageString multiLanguageString = getMultiLanguageString(entity);
+        Map<String, String> mlsMap = new HashMap<>(multiLanguageString.data());
+        mlsMap.put("fallback", values.at(0).getString());
```
Comment (meg1502) [inline src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java:214]: what about MLS fields with fallback disabled?

Comment (idlira) [reply src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java:214]: Worse case we have to look at this after the patch. The highest priority now is to bring a working version in OX for the Content entities and they all have fallback enabled.

**Review by sabieber — APPROVED** [review]

File: src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java (hunk 2 above)
Comment (cschierle) [reply src/main/java/sirius/db/mixing/properties/MultiLanguageStringProperty.java]: Treating MLS fields without fallback is fixed in https://github.com/scireum/sirius-db/pull/405: https://github.com/scireum/sirius-db/pull/405/commits/ff2bacd66ec9cb0ad1aa713543558e45043f0edf

**Review by meg1502 — APPROVED** [review]

**Comment by cschierle** [comment]
...also these changes are included in #404 🤷

**Review by idlira — APPROVED** [review]

**Comment by cschierle** [comment]
changes are included in #404

**Pull request closed without merging**